### DISCUSSION TITLE
blog/how-to-build-istio/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#281] [Announcements]
# blog/how-to-build-istio/

本文将指导你如何在 macOS 上编译 Istio。

https://jimmysong.io/blog/how-to-build-istio/

**Comment by rootsongjc** [comment]
这篇博客是有人在 Istio slack 上询问 Istio 编译问题而撰写的，我在个人电脑上编译没有出现问题。
